src/bebone/assets/image/image_painter.h

```diff
@@ -46,6 +61,14 @@ namespace bebone::assets {
             */
             void paint_square(const size_t& x_pos, const size_t& y_pos, const f32& sq_width, const f32& sq_height, const Color& color);
 
+            /*!
+            * Draws square to the image
+            * @param pos - Square bottom left corner position
+            * @param size - Square size
+            * @param color - Square color
+            */
+            void paint_square(const Vec2i& pos, const Vec2i & size, const Color& color);
```

Comment:
Change `const Vec2i & size` to `const Vec2i& size`

Reply:
Ok boss !